chart/Chart.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v2
+appVersion: latest
```

This is the latest image in GCR.

```suggestion
appVersion: 0.2.0
```